chasm/tree.go

```diff
@@ -727,8 +771,12 @@ func (n *Node) syncSubField(fieldV reflect.Value, fieldN string, nodePath []stri
 		// Field is not empty but tree node is not set. It means this is a new field, and a node must be created.
 		childNode := newNode(n.nodeBase, n, fieldN)
 
-		if err = assertStructPointer(reflect.TypeOf(internal.value())); err != nil {
-			return
+		switch internal.value().(type) {
+		case []string: // Pointer
```

Comment:
shall we also check if fieldType is pointer here?

Reply:
For some reason I thought that `fieldType` might be not set there. But it should be always set. Changing to switch on `fieldType` because it is much better.